great_expectations/validation_operators/validation_operators.py

```diff
@@ -19,6 +19,42 @@
 # NOTE: Abe 2019/08/24 : This is first implementation of all these classes. Consider them UNSTABLE for now.
 
 
+class ValidationOperatorResult(object):
+    # TODO this is a design sketch
+    def __init__(self):
+        raise NotImplementedError
+
+    def get_run_id(self):
+        raise NotImplementedError
+
+    def list_assets_validated(self) -> list:
+        raise NotImplementedError
+
+    def get_evaluation_parameters(self):
+        raise NotImplementedError
+
+    @property
+    def statistics(self) -> dict:
+        raise NotImplementedError
+
+    @property
+    def actions_results(self) -> dict:
+        raise NotImplementedError
+
+    @property
+    def success(self) -> bool:
+        raise NotImplementedError
+
+    @property
+    def details(self):
+        # TODO I'm not sure what type this should return
+        raise NotImplementedError
+
+    @property
+    def validation_result_by_expectation_suite_identifier(self) -> dict:
```

Comment:
I like just "validation_results"

Reply:
Ahh, never mind...I see. Is this a mapping from suite name to actual result or to the id of the result?

Reply:
tbh I'm not sure yet.

Reply:
A suite can appear more than once in one call to a validation operator run - e.g., it can be used to validated the batches of 2 data assets). This can break this dictionary.